rfcs/writing-cross-platform-typescript.md

```diff
@@ -0,0 +1,134 @@
+# Writing Cross Platform TypeScript 
```

Comment:
Let's link this in contributing guide too

Reply:
Will do.

Reply:
Done.